qiskit/aqua/algorithms/classifiers/qsvm/qsvm.py

```diff
@@ -134,14 +134,13 @@ def __init__(self, feature_map: Union[QuantumCircuit, FeatureMap],
             self.feature_map_params_x = ParameterVector('x', self.feature_map.feature_dimension)
             self.feature_map_params_y = ParameterVector('y', self.feature_map.feature_dimension)
         else:
-            if not isinstance(feature_map, RawFeatureVector):
-                warnings.warn("""
-                The {} object as input for the QSVM is deprecated as of 0.7.0 and will
-                be removed no earlier than 3 months after the release.
-                You should pass a QuantumCircuit object instead.
-                See also qiskit.circuit.library.data_preparation for a collection
-                of suitable circuits.""".format(type(feature_map)),
-                              DeprecationWarning, stacklevel=2)
+            warnings.warn("""
+            The {} object as input for the QSVM is deprecated as of 0.7.0 and will
```

Comment:
Should this text be updated to 0.9.0 (otherwise with 0.7.0 it seems support logically should have ended long ago)

Reply:
Absolutely, thanks!